test/e2e/utils.go

```diff
@@ -157,3 +162,28 @@ func getSecurityContextRestrictedProfile() *v1.SecurityContext {
 		},
 	}
 }
+
+func getOrCreateCMOConfig(t *testing.T) (*v1.ConfigMap, error) {
```

Comment:
I didn't have a look at all the recent changes, but this drew my attention.
are the existing `BuildCMOConfigMap` and `MustCreateOrUpdateConfigMap` not sufficient for this?

Reply:
Right, we can use `BuildCMOConfigMap` to construct the ConfigMap [here](https://github.com/openshift/cluster-monitoring-operator/pull/2283/files#diff-2a8a052d42916d916acab3eed2d52f7039647cc165e3086304d8c4e3e3b800bcR172-R180). 👍🏼